modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/images/PodcastImageLoader.kt

```diff
@@ -57,6 +57,16 @@ open class PodcastImageLoader(
         }
     }
 
+    suspend fun getBitmapSuspend(podcast: Podcast, size: Int): Bitmap? {
+        return try {
+            val request = load(podcast).size(size, size).build()
+            context.imageLoader.execute(request).drawable!!.toBitmap()
```

Comment:
Is the reason we're using `!!` here instead of `?` like we are on line 66 because if it returns null we'd rather fallback to the default podcast image than return `null`? What would you think about adding a short comment explaining that? It wasn't obvious to me at first glance why we were using `!!`.

Reply:
This logic was copied over from the existing methods on lines 36 and 48. I didn't look too deep into it but that seems to be the intended operation - a fallback mechanism. I can add a comment clarifying, but that also means I'd have to do the same for the other two methods.
Alternatively, we could change the logic to use an `if-else` instead of a`try-catch` with a generic exception for control flow. Something like this:
```
val request = load(podcast).size(size, size).build()
val drawable = context.imageLoader.execute(request).drawable
return if (drawable != null) {
        drawable.toBitmap()
    } else {
        val fallbackRequest = loadNoPodcastCoil().size(size, size).build()
        context.imageLoader.execute(fallbackRequest).drawable?.toBitmap()
    }
```

Reply:
I really like your if-else suggestion. If you want to make that change, that would be great.

Reply:
I'll work on this change.

Reply:
Done @mchowning 
Please review.